packages/web3torrent/tracker/index.js

```diff
@@ -1,12 +1,27 @@
 const Server = require('bittorrent-tracker').Server;
 const log = require('debug')('web3torrent:tracker');
-const port = process.env.TRACKER_PORT || 8000;
+const isAddress = require('web3').utils.isAddress;
+const port = process.env.PORT || 8000;
+
+const ACTIONS = {CONNECT: 0, ANNOUNCE: 1, SCRAPE: 2, ERROR: 3};
+const EVENTS = {START: 'started', COMPLETE: 'completed', UPDATE: undefined, STOP: undefined};
 
 const server = new Server({
   udp: true,
   http: true,
   ws: true,
-  stats: true
+  stats: true,
+  filter: function(_, params, cb) {
+    if (params && params.action === ACTIONS.ANNOUNCE && params.event === EVENTS.START) {
```

Comment:
Should this throw an error if `params` is `null` or `undefined`?

Reply:
Are there actions besides `ACTIONS.ANNOUNCE` and events besides `EVENTS.START` that do not require pseAccount to be set?

Reply:
Yeah, actions has **update**, **scrape** and **error** (I know that **scrape** is used, I've never seen when **error** is thrown).
Events has **complete**, **update** and **stop**.

Announce is used when starting to seed, and when starting to leech.